std_msgs/Float64MultiArray joints
---
bool success
